frontend/src/components/AdminSuspendModal/AdminSuspendModal.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Button, Modal, ModalHeader, ModalBody, ModalFooter,
+} from 'reactstrap';
+import './AdminSuspendModal.css';
+import { suspendUser } from '../../lib/apiClient';
+
+class AdminSuspendModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const {
+      toggleSuspendModal, suspendModal, selectedUser, refreshUsers,
+    } = this.props;
+
+    const handleSuspend = () => {
+      suspendUser(selectedUser, selectedUser.id).then(() => {
```

Comment:
Maybe instead of sending the whole user and handling the toggling in the API call, we actually just send the user ID and `true` or `false` to `suspendUser`? That way we can keep the actual toggling logic here where it's being done, and the `suspendUser` can just submit what we tell it to submit.

Reply:
I'd still love to fix this function call, to something like `suspendUser(selectedUser.id, !selectedUser.is_suspended)`, and change the `suspendUser` function in `apiClient` to just submit the second argument instead of toggling.